### PR TITLE
cmake: Use common LunarG function for build config

### PR DESCRIPTION
**Pull request by christophe-lunarg**
A PR mainly also to try to understand the issue Jeremy experienced...

### CODE REVIEW

**Comment by christophe-lunarg** [comment]
> > A PR mainly also to try to understand the issue Jeremy experienced...
> 
> Jeremy was having issues due to the utility library exporting strict compiler warnings: [KhronosGroup/Vulkan-Utility-Libraries#103](https://github.com/KhronosGroup/Vulkan-Utility-Libraries/issues/103)

Yes, I am resolving the issues in the repository so that the inheritance of the `Vulkan-Utility-Libraries` build options don't cause issues anymore. 

If that's a good plan with you.

**Comment by juan-lunarg** [comment]
> If that's a good plan with you.

That's a good plan 👍🏾

File: CMakeLists.txt [hunk 1]
```diff
@@ -19,6 +19,8 @@ cmake_minimum_required(VERSION 3.17.2)
 
 project(VEL LANGUAGES CXX)
 
+option(BUILD_TESTS "Build tests")
+
```
Comment (juan-lunarg) [inline CMakeLists.txt:23]: Why was this changed?

Comment (christophe-lunarg) [reply CMakeLists.txt:23]: That was to be sure that the option exist because it's used.... but maybe it's not how conceptionally it works. 
`BUILD_TESTS` now used when lunarg_target_compiler_configurations is called.

Comment (juan-lunarg) [reply CMakeLists.txt:23]: Why not use `BUILD_WERROR`?

**Comment by jeremyg-lunarg** [comment]
Is this PR intended to be merged or just a test?

File: CMakeLists.txt (hunk 1 above)
Comment (christophe-lunarg) [reply CMakeLists.txt:23]: Oups, mistake!

Comment (christophe-lunarg) [reply CMakeLists.txt]: Fixed!

**Comment by christophe-lunarg** [comment]
> Is this PR intended to be merged or just a test?

Yes this is intented to be merged.

File: CMakeLists.txt [hunk 2]
```diff
@@ -19,6 +19,8 @@ cmake_minimum_required(VERSION 3.17.2)
 
 project(VEL LANGUAGES CXX)
 
+option(BUILD_WERROR "Treat compiler warnings as errors")
```
Comment (juan-lunarg) [inline CMakeLists.txt:22]: Revert the change to BUILD_WERROR. It will be treated as false if it isn't defined. Which works out since the default value is OFF.

This same logic is used for BUILD_TESTS. See scripts/CMakeLists.txt

**Review by juan-lunarg — APPROVED** [review]

File: CMakeLists.txt (hunk 2 above)
Comment (christophe-lunarg) [reply CMakeLists.txt:22]: Maybe but I guess / hope it's going to change.

**Review by juan-lunarg — APPROVED** [review]
